stripper/ze_Ancient_wrath_r4.cfg

```diff
@@ -54,4 +54,22 @@ modify:
 		"OnStartTouch" "anticamp_hurtEnable5.51"
 		"OnStartTouch" "anticamp_hurtDisable8.51"
 	}
-}
\ No newline at end of file
+}
+
+;nerf cancer boss hp 
+modify:
+{
+	match:
+	{
+		"classname" "trigger_multiple"
+		"targetname" "boss_2_player_counter_trigger"
+	}
+	delete:
+	{
+		"OnStartTouch" "boss_3_knockback_1_counterAdd4350-1"
+	}
+	insert:
+	{
+		"OnStartTouch" "boss_3_knockback_1_counter,Add,240,0,-1"
+	}
+}
```

Comment:
I don't recall there being anything wrong with the HP scaling on this boss?

Reply:
although there is a max HP of 13000, it still makes it RNG due to the boss being moved, when we played it most of the time, we never got past the boss because of the HP, when we did we would almost always lose the round due to how many players would be alive.

Reply:
I don't think changing the HP is necessary, the map has been on the server for 2 years with plenty of wins. I personally have not experienced "most of the time, we never got past the boss because of the HP" as you claim. Sure we fail the boss or occasionally get stuck there, but that's just natural to have happen sometimes.